tarantoolctl

```diff
@@ -0,0 +1,1353 @@
+#!/usr/bin/env tarantool
```

Comment:
TBH, I don't see where test-run is changed to use this `tarantoolctl` executable.

Reply:
I see the related changes in `find_exe()` now.